src/api/entities/TickerReservation/index.ts

```diff
@@ -188,10 +186,9 @@ export class TickerReservation extends Entity<UniqueIdentifiers, string> {
    * Transfer ownership of the Ticker Reservation to another Identity. This generates an authorization request that must be accepted
    *   by the destinatary
    *
-   * @note this will create [[AuthorizationRequest | Authorization Requests]] which have to be accepted by
-   *   the corresponding [[Account | Accounts]] and/or [[Identity | Identities]].
-   *   An Account or Identity can fetch its pending Authorization Requests by calling {@link Authorizations.getReceived}.
-   *   Also, an Account or Identity can directly fetch the details of an Authorization Request by calling {@link Authorizations.getOne}
+   * @note this will create {@link AuthorizationRequest Authorization Request} which has to be accepted by the `target` Identity.
```

Comment:
```suggestion
   * @note this will create {@link AuthorizationRequest | Authorization Request} which has to be accepted by the `target` Identity.
```